src/Administration/Resources/app/administration/src/app/component/form/sw-custom-field-set-renderer/sw-custom-field-set-renderer.html.twig

```diff
@@ -71,21 +71,21 @@
             ref="tabComponent"
             class="sw-custom-field-set-renderer__card-tabs"
             variant="minimal"
-            :default-item="visibleCustomFieldSets[0].name"
+            :default-item="visibleCustomFieldSets[0].id"
             :position-identifier="'sw-custom-field-set-renderer'"
+            @new-item-active="(tab) => loadCustomFieldSet(tab.name)"
         >
             {% block sw_custom_field_set_renderer_card_tabs %}
-            <template slot-scope="{ active }">
+            <template #default="{ active }">
                 <template v-for="set in visibleCustomFieldSets">
                     <sw-tabs-item
-                        v-if="set.customFields"
                         :key="`sw-tab--${set.id}`"
                         variant="minimal"
-                        :name="set.name"
+                        :name="set.id"
```

Comment:
Why though?

Reply:
everything was changed to ids

Reply:
The prop `name` is abused to hold an `id`. Now I have something that depends on this being a name and everything breaks. This is a breaking change, guys :/

Reply:
Thank you for looking into it again. 

I am not really depending on the name, but I overwrite how the name is displayed and for this I had to copy a section you changed. My template is now fixed but I did not have a better entrypoint :/ see:

```
{% block sw_custom_field_set_renderer_card_tabs %}
    <template slot-scope="{ active }">
        <template v-for="set in visibleCustomFieldSets">
            <sw-tabs-item
                v-if="set.customFields"
                :key="`sw-tab--${set.id}`"
                variant="minimal"
                :name="set.id"
                :class="'sw-tab--name-' + set.name"
                :active-tab="active"
            >
                {{ ((set.config.label.snippet && $te(set.config.label.snippet)) ? $t(set.config.label.snippet) : getInlineSnippet(set.config.label)) || set.name }}
            </sw-tabs-item>
        </template>
    </template>
{% endblock %}
```

Will soon be a PR :)